pkg/prompts/prompts.go

```diff
@@ -79,6 +87,17 @@ func RunPromptsFromConfigWithSkipsIO(config *config.DraftConfig, varsToSkip []st
 // GetVariableDefaultValue returns the default value for a variable, if one is set in variableDefaults from a ReferenceVar or literal VariableDefault.Value in that order.
 func GetVariableDefaultValue(variableName string, variableDefaults []config.BuilderVarDefault, inputs map[string]string) string {
 	defaultValue := ""
+
```

Comment:
looks good
can we move this to createDeployment and override the default value in the deployConfig.variableDefaults?
otherwise this check will be running for every variable substitution, not just the deployment app name

Reply:
Actually I think this is the correct place for now. This default is for CLI input only, not all template generation input so it wouldnt make sense to put it in the draft config. Also along those lines, how would we show a dynamic value as a "Default" value, I dont think it would make sense